src/OpenSSL/version.py

```diff
@@ -11,7 +11,7 @@
     "__title__", "__uri__", "__version__",
 ]
 
-__version__ = "0.16.dev0"
+__version__ = "15.2.0.dev0"
```

Comment:
Yep, I've discussed that on #cryptography-dev.   SemVer just doesn't make any sense for pyOpenSSL.